src/Backend/Modules/Blog/DataFixtures/LoadBlogCategories.php

```diff
@@ -2,29 +2,41 @@
 
 namespace Backend\Modules\Blog\DataFixtures;
 
+use Backend\Core\Language\Locale;
+use Backend\Modules\Blog\Domain\Category\Category;
+use Common\Doctrine\Entity\Meta;
 use SpoonDatabase;
+use Symfony\Bundle\FrameworkBundle\Client;
 
 class LoadBlogCategories
 {
+    /**
+     * @var Client
+     */
+    private $client;
+
+    public function __construct(Client $client)
```

Comment:
maybe limit this to just the container, that way you still have the flexibility but not the overhead, though it would be cleaner to only pass the services you actually need

Reply:
fixed in d57655357519c0491d8677a1792bb30174836f19